plyer/platforms/linux/screenshot.py

```diff
@@ -0,0 +1,29 @@
+import subprocess
+from plyer.facades import ScreenShot
+from plyer.utils import whereis_exe
+
+
+class GnomeScreenShot(ScreenShot):
```

Comment:
Perhaps a reference to the options (gnome, import) might be useful in the documentation?